db_schemas/V001__initialization.sql

```diff
@@ -16,9 +19,18 @@ drop table contact_tracer.device_location;
 create table if not exists contact_tracer.device_location (
 		device_id varchar not null references contact_tracer.device(device_id),
 		sample_date timestamp,
-		location geometry,
+		location postgis.geometry,
```

Comment:
this is what you recreated right?

Reply:
this and the thing at postgis extension created at the top

Reply:
Thanks!